.buildkite/commands/run-ui-tests.sh

```diff
@@ -24,6 +24,19 @@ install_gems
 echo "--- :cocoapods: Setting up Pods"
 install_cocoapods
 
+echo "--- :test-analytics: Configure Test Analytics"
+# Collect data separately for iPhone and iPad
+if [[ $DEVICE =~ ^iPhone ]]; then
+  export BUILDKITE_ANALYTICS_TOKEN=$BUILDKITE_ANALYTICS_TOKEN_UI_TESTS_IPHONE
+else
+  export BUILDKITE_ANALYTICS_TOKEN=$BUILDKITE_ANALYTICS_TOKEN_UI_TESTS_IPAD
+fi
+
+buildkite-agent annotate \
+  'Test Analytics for UI tests are currently unavailable' \
```

Comment:
I just realized that the wording for this might be confusing — especially for devs not familiar with TA nor with your struggles while working on it.

See https://github.com/wordpress-mobile/WordPress-iOS/pull/19010#discussion_r916564202 for wording change suggestions.


TL;DR one suggestion is having a GitHub issue tracking this with more info about what that means and add a link to it, another is just change the wording to this:

```suggestion
  "Buildkite Test Analytics are currently not reported for UITests. We're investigating why and working on fixing it." \
```

Or you can have a mix of both (ie wording above + link to issue), up to you.

Reply:
It turns out that the issue I thought was only due to my UI tests setup is an issue with the Buildkite library and APIs, see https://github.com/buildkite/test-collector-swift/issues/25.

So, I might sit on this for a few days and hopefully I can just remove the notice altogether.